config/phenomenal_tools2container.yaml

```diff
@@ -76,15 +76,6 @@ assignment:
   max_pod_retrials: 3
   tools_id:
   - isa2w4m
-
```

Comment:
Why is this going away?

Reply:
Ok, I see downwards that you are deleting wrappers and other entries, so not a merging error that removed this.